ddp-workspace/projects/ddp-pancan/src/app/components/faq-section/faq-section.component.html

```diff
@@ -0,0 +1,22 @@
+<mat-accordion hideToggle="true" multi="true" displayMode="flat">
+    <ng-container *ngFor="let item of questions | translate; let i=index;">
+        <ng-container
+            *ngTemplateOutlet="question; context: { item: item, show: (questionsCount && i < questionsCount) || !questionsCount }">
```

Comment:
#minor you can use slice pipe to limit ngFor https://angular.io/api/common/SlicePipe

Reply:
good idea, thanks!
Fixed at 5934f4245f5e114a6153ffdb7e957e24f6eb646a